apps/web/src/components/ProjectsGrid/index.tsx

```diff
@@ -11,19 +11,25 @@ interface ProjectsGridProps {
   onRefetch: () => void;
   loading: boolean;
   nextCursor: string | null;
+  previous?: string;
 }
 
 const ProjectsGrid = ({
   projects,
   loading,
   onRefetch,
   nextCursor,
+  previous = '/',
 }: ProjectsGridProps) => {
   return (
     <>
       <StyledProjectsGrid>
         {projects.map((project) => (
```

Comment:
Should we also handle empty results, like maybe displaying a message?

Reply:
true, I'll add something